Open Judge System/Web/OJS.Web/Areas/Contests/Views/Results/Simple.cshtml

```diff
@@ -38,120 +32,13 @@
-                    <td>@participant.ParticipantFullName</td>
-                    @foreach (var participantResult in participant.ProblemResults)
-                    {
-                        if (User.IsAdmin() || Model.UserIsLecturerInContest)
-                        {
-                            if (participantResult.BestSubmission == null)
-                            {
-                                if (!participantResult.IsPartOfUserProblems &&
-                                    Model.IsCompete &&
-                                    Model.ContestType == ContestType.OnlinePracticalExam)
-                                {
-                                    <td>-</td>
-                                }
-                                else
-                                {
-                                    <td>0</td>
-                                }
-                            }
-                            else
-                            {
-                                <td><a href="/Contests/Submissions/View/@participantResult.BestSubmission.Id">@participantResult.BestSubmission.Points</a></td>
-                            }
-                        }
-                        else if (participantResult.ShowResult)
-                        {
-                            if (participantResult.BestSubmission == null)
-                            {
-                                if (!participantResult.IsPartOfUserProblems &&
-                                    Model.IsCompete &&
-                                    Model.ContestType == ContestType.OnlinePracticalExam)
-                                {
-                                    <td>-</td>
-                                }
-                                else
-                                {
-                                    <td>0</td>
-                                }
-                            }
-                            else
-                            {
-                                <td>@participantResult.BestSubmission.Points</td>
-                            }
-                        }
-                    }
-                    @if (User.IsAdmin() || Model.UserIsLecturerInContest)
-                    {
-                        <td>@participant.AdminTotal</td>
-                    }
-                    else
-                    {
-                        <td>@participant.Total</td>
-                    }
-                </tr>
-            }
-        </tbody>
-    </table>
-}
-else
-{
-    <h2>@Resource.NoMoreResults</h2>
-}
-
-@if ((Model.CurrentPage == 1 && results.Count == resultsOnPage) || Model.CurrentPage > 1)
-{
-    @Html.Partial("_SimplePagination", new SimplePaginationViewModel { HasNext = results.Count == resultsOnPage, CurrentPage = Model.CurrentPage, Url = "/Contests/" + isOfficial + "/Results/Simple/" + Model.Id + "?page=" })
-}
\ No newline at end of file
+@Html.Partial("_SimplePartial", Model)
```

Comment:
The name of the partial should be more expressive, currently its unclear what "_SimplePartial" is exactly? Rename to something like "\_SimpleResultsPagedList" ( the "\_" at the start of the view name means its a partial view, so there is no need to manually write it in the name)

Reply:
Done in https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/223